README.md

```diff
@@ -3843,6 +3845,31 @@ also known as
 cat .git/config
 is a useful command by the way
 
+<<<<<<< HEAD
```

Comment:
Now it also have this <<<<< >>>>> lines - they should be removed when you resolving conflicts ;)

Reply:
Hey,  chief dude, have I done everything?))